guilded/client.py

```diff
@@ -198,10 +199,10 @@ async def listen_socks(ws, team=None):
                     except WebSocketClosure as exc:
                         code = ws._close_code or ws.socket.close_code
                         if teamId:
-                            log.warning(f'Team {teamId}\'s websocket closed with code {code}, attempting to reconnect in {next_backoff_time} seconds')
+                            log.warning('Team %s\'s websocket closed with code %s, attempting to reconnect in %s seconds', teamId, code, next_backoff_time)
                             self.dispatch('team_disconnect', teamId)
                         else:
-                            log.warning(f'Websocket closed with code {code}, attempting to reconnect in {next_backoff_time} seconds')
+                            log.warning('Websocket closed with code %s, attempting to reconnect in %s seconds', code, next_backoff_time)
```

Comment:
Is this faster than f-strings? Not exactly more readable imo, just the opposite.

Reply:
Loggers are designed work like this, not with f-string. If anything happens during formatting message it will only result in exception inside logger that will be handled. f-string also prevent in aggregation of such cases.